CHANGES.md

```diff
@@ -29,6 +29,8 @@
 - Parsing support has been added for walruses inside generator expression that are
   passed as function args (for example,
   `any(match := my_re.match(text) for text in texts)`) (#3327).
+- Reformat empty and whitespace-only files as either an empty file (if no newline is
```

Comment:
This needs to go into the preview style (not just in the changelog, but also in the code).

Reply:
Should this be added in the future style page too?